src/main/kotlin/io/github/tsb99x/trakt/Utils.kt

```diff
@@ -1,6 +1,17 @@
 package io.github.tsb99x.trakt
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.sql.ResultSet
 import java.util.*
 
+const val REQUEST_ID = "requestId"
+
 fun String.toUUID(): UUID =
     UUID.fromString(this)
+
+fun ResultSet.getUUID(columnLabel: String): UUID =
+    this.getObject(columnLabel, UUID::class.java)
```

Comment:
Just a proper way to retrieve UUID from ResultSet. No need to use `ResultSet.getString(columnLabel).toUUID()`.

Reply:
Also a good idea is to provide additional extension function for LocalDateTime.

```suggestion
    fun ResultSet.getLocalDateTime(columnLabel: String): LocalDateTime =
        this.getTimestamp(columnLabel).toLocalDateTime()
```